pkg/apis/monitoring/v1alpha1/alertmanager_config_types.go

```diff
@@ -102,8 +105,29 @@ type Route struct {
 	// Boolean indicating whether an alert should continue matching subsequent
 	// sibling nodes. It will always be overridden to true for the first-level
 	// route by the Prometheus operator.
-	Continue bool    `json:"continue,omitempty"`
-	Routes   []Route `json:"routes,omitempty"`
+	Continue bool `json:"continue,omitempty"`
+	// Child routes.
+	Routes []apiextensionsv1.JSON `json:"routes,omitempty"`
+	// Note: this comment applies to the field definition below but appears
```

Comment:
> Note: this comment applies to the field definition below but appears below otherwise it gets included in the generated manifest.

Not sure I understand the two below things, maybe we can clarify this a bit?

Reply:
you're right, it didn't read correctly :)